website/docs/Use-Cases/Auto-Generation.md

```diff
@@ -380,6 +380,8 @@ The compact history is more efficient and the individual API call history contai
 
 [`flaml.autogen.agents`](../reference/autogen/agent/agent) contains an experimental implementation of interactive agents which can adapt to human or simulated feedback. This subpackage is under active development.
 
+Find a notebook example on how to [use agents in FLAML to solve coding tasks](https://github.com/microsoft/FLAML/blob/main/notebook/autogen_agent.ipynb).
```

Comment:
solve coding tasks -> perform tasks with code?

Reply:
Also change the text and format to match other links

Reply:
I have another PR updating this documentation. Can I change this in the other PR?

Reply:
How should I add notebook links in #1056 ?

Reply:
Changed based on my understanding of "match other links."